examples/src/kotlin/org/partiql/examples/ParserExample.kt

```diff
@@ -45,7 +45,9 @@ class ParserExample(out: PrintStream) : Example(out) {
         // version of the s-expression form to an instance of [ExprNode].
         val deserializedAst = serializedAst.toExprNode(ion)
         // Verify that we have the correct AST.
-        assertEquals(originalAst, deserializedAst)
+        if (originalAst != deserializedAst) {
+            throw Exception("expected AST to be equal")
+        }
```

Comment:
Could we instead use the `assertEquals` with a message parameter for the exception? I am not sure what we are gaining by using `if` and throwing our own exception here, perhaps I am missing something though.

Reply:
The reason was to avoid requiring the `kotlin-test` JAR from code that is technically not an automated unit test. I will revert that part of the change since you would prefer to use this dependency.

Reply:
Thanks for explaining. In that case, ignore my comments for this file on using `assert`. Let's keep the examples clean of testing libs.